test/fixtures/v2/documentRopstenInvalidWithDIDSignedProofBlockProperHash.ts

```diff
@@ -0,0 +1,55 @@
+import { v2, WrappedDocumentWithProof } from "@govtechsg/open-attestation";
```

Comment:
what is invalid ?

Reply:
The `signature` is invalid. Changed the first 3 chars of the `signature` to `123` to be used for failing cases in the integration tests.

Reply:
can you add a comment somewhere in this file ?

Reply:
yessir added a comment above where the document is defined.
```
/**
 * This document is invalid as the signature within the proof block has been tampered with.
 * The first 3 characters have been changed to 123. To be used for integration testing.
 */
```